crates/router/src/core/errors/api_error_response.rs

```diff
@@ -92,7 +92,7 @@ pub enum ApiErrorResponse {
     AccessForbidden,
     #[error(error_type = ErrorType::InvalidRequestError, code = "IR_23", message = "{message}")]
     FileProviderNotSupported { message: String },
-    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_23", message = "{entity} expired or invalid")]
+    #[error(error_type = ErrorType::InvalidRequestError, code = "IR_23", message = "{entity}")]
     UnprocessableEntity { entity: String },
```

Comment:
```suggestion
    UnprocessableEntity { entity: String, message: String},
```
Sending the whole message in entity does not seem right, you can split these into entity and message